### ISSUE TITLE
Node resources utilisation vs pod utilisation 

### ISSUE THREAD

**Issue by vuravenkatachaitanya** [kubernetes/kubernetes#119797]
### What would you like to be added?

Resource Configuration: The pod has been thoughtfully configured with resource limits and requests to ensure efficient resource allocation.

Misleading Node Utilization: Initial analysis of node utilization indicated that resource requests were over 200%, while limits were coming at 99%. This might suggest high utilization at first glance.

Actual Utilization Unveiled: However, a closer look at actual resource utilization revealed that the pods are, in fact, underutilized. This has led to node utilization remaining relatively low, not exceeding 37%. This insight came from monitoring OS commands such as 'top' command for resource utilization.

Resource Allocation Discrepancy: It appears that despite the underutilization of pods, the resource allocation for these pods on the node is disproportionately high.

Failed Pod Assignments: This skewed resource allocation prevents the node from accepting new pods. Attempts to schedule new pods are resulting in failures with the error message: "0/2 nodes are available: 2 Insufficient memory nodes."

In summary, despite the seemingly high request and limit values, the actual resource utilization of the pods is lower than expected. The uneven allocation of resources for these pods is causing node utilization to remain low and new pod assignments to fail due to insufficient memory.

### Why is this needed?

- Trigger are set for autoscaling which never hits because of under utilised server or provision new pods in existing nodes.

**Comment by vuravenkatachaitanya** [comment]
@tzneal Thanks for your reply. 

My bad i was talking vice versa of requests and limits :
```
 (Total limits may be over 100 percent, i.e., overcommitted.)
  Resource                    Requests          Limits
  --------                    --------          ------
  cpu                         3115m (41%)       11400m (152%)
  memory                      9140490624 (29%)  20921295616 (66%)
```
I could not able to recreate the issue now. But I have seen cases where mapr pod (memory intensive) applications used to take over the node resources completely and starve for other pods or fails existing pods.

But convincing our capacity team about the node utilisation where they take utilisation report very conventionally using  vmstat and top dose not shows load not more than 35% and they don't agree to release new nodes for eks clusters.

**Comment by tzneal** [comment]
There are docs at https://kubernetes.io/docs/concepts/configuration/manage-resources-containers that cover requests/limits. https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/ is probably also of interest since it covers QoS and how its affected by requests/limits.


/kind support

**Comment by neolit123** [comment]
/sig node apps
/close

you could join the above SIG meetings or their slack channels for more direct feedback:
https://github.com/kubernetes/community/tree/master/sig-node/README.md
https://github.com/kubernetes/community/tree/master/sig-apps/README.md

if they confirm there is a missing feature or something is not behaving as designed, please re-open this ticket.
